lib/src/validation-server/spots/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { String } from "../../lib";
+
+export interface BaseError {
+  error_code: String;
+  error_messages: String[];
+}
+
+export interface UnprocessableEntityError extends BaseError {
+  type: "unprocessable_entity";
+}
+
+export type Headers = String[];
```

Comment:
For a stronger contract we can consider something like:
```suggestion
export type Headers = Header[];
interface Header = {
  name: String;
  value: String;
}
```

Reply:
Great suggestion, sounds obvious now 😂